### PR TITLE
Fix ffmpeg call on linux

### PR DESCRIPTION
**Pull request by bratkartoff**
Popen() with strings only works on windows or with shell=True, using lists works on linux as well (and might prevent errors with weird filenames).
I haven't tested this on windows.

### CODE REVIEW

File: jumpcutter.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#!/bin/env python3
```
Comment (Gegell) [inline jumpcutter.py]: Is this line truly needed for running on linux? Because for my windows installation this screws me over, as now the python launcher py.exe tries to look in the specified directory (which does not exist) and thus fail to run.

File: jumpcutter.py [hunk 2]
```diff
@@ -249,12 +254,20 @@ def speed_up_video(
...
+    command = [
+        '-i', input_file,
+        '-i', temp_folder + '/audioNew.wav',
+        '-filter_script:v', temp_folder + '/filterGraph.txt',
+        '-map', '0',
+        '-map', '-0:a',
+        '-map', '1:a',
+        '-c:a', 'aac',
+        output_file,
+        '-loglevel', 'warning'
```
Comment (Gegell) [inline jumpcutter.py]: missing a trailing `,` here. This causes the next `-stats` to be concatenated to the previous string, which causes issues with running ffmpeg.

File: jumpcutter.py [hunk 3]
```diff
@@ -34,8 +35,8 @@ def _is_valid_input_file(filename) -> bool:
     :return: True if it is a file with an audio stream attached.
     """
 
-    command = 'ffprobe -i "{}" -hide_banner -loglevel error -select_streams a' \
-              ' -show_entries stream=codec_type'.format(filename)
+    command = ['ffprobe', '-i', filename] + '-hide_banner -loglevel error -select_streams a' \
+              ' -show_entries stream=codec_type'.split(' ')
```
Comment (Gegell) [inline jumpcutter.py]: Personally I'd lie the command creation to be more consistent, e.g. at this point just put everything into a list & split the command manually. Just so that it's more consistent with the command creation below.

File: jumpcutter.py (hunk 1 above)
Comment (Gegell) [reply jumpcutter.py]: So I looked over it, and apparently the launcher has some default behavior for *some* of the shabang lines.
See https://docs.python.org/3/using/windows.html#shebang-lines. While it also mentions that one could change the settings of the py launcher, I want this script to work out of the box (on windows) without weird edge cases on the launching behaviour.

tldr: For `#!/usr/bin/env` the py.exe launcher actually also looks through the windows PATH env variable whether it finds a suitable python installation. However since by default those are only named `python.exe` and not `python3.exe` we would hence need to change the line to:
```suggestion
#!/usr/bin/env python
```

While this removes the constraint of the file needing python 3, I believe this should be a good compromise over outright removing the line :)

File: jumpcutter.py (hunk 2 above)
Comment (Gegell) [reply jumpcutter.py]:
```suggestion
        '-loglevel', 'warning',
```

File: jumpcutter.py [hunk 4]
```diff
@@ -151,8 +152,8 @@ def speed_up_video(
     _create_path(temp_folder)
 
     # Find out framerate and duration of the input video
-    command = 'ffprobe -i "{}" -hide_banner -loglevel error -select_streams v' \
-              ' -show_entries format=duration:stream=avg_frame_rate'.format(input_file)
+    command = ['ffprobe', '-i', input_file] + '-hide_banner -loglevel error -select_streams v' \
+              ' -show_entries format=duration:stream=avg_frame_rate'.split(' ')
```
Comment (Gegell) [inline jumpcutter.py]:
```suggestion
    command = [
        'ffprobe', '-i', input_file, '-hide_banner', '-loglevel', 'error',
        '-select_streams', 'v', '-show_entries', 'format=duration:stream=avg_frame_rate'
    ]
```

File: jumpcutter.py (hunk 3 above)
Comment (Gegell) [inline jumpcutter.py]:
```suggestion
    command = [
        'ffprobe', '-i', filename, '-hide_banner', '-loglevel', 'error',
        '-select_streams', 'a', '-show_entries', 'stream=codec_type'
    ]
```
